boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -44,6 +50,83 @@
 		led0 = &green_led_4;
 		sw0 = &user_button;
 	};
+
+	fmc: memory-controller@A0000000 {
```

Comment:
```suggestion
	fmc: memory-controller@a0000000 {
```

addresses are usually written in lower-case

Reply:
Got it!